Doc/library/pprint.rst

```diff
@@ -217,9 +217,10 @@ let's fetch information about a project from `PyPI <https://pypi.org>`_::
    >>> import json
    >>> import pprint
    >>> from urllib.request import urlopen
-   >>> with urlopen('http://pypi.org/project/Twisted/json') as url:
+   >>> with urlopen('https://pypi.org/pypi/Twisted/json') as url:
```

Comment:
Output of this URL is changed, and it's too large for example.  (Output is in below).
I think we should find better json example.

Reply:
Maybe we can add a new JSON file in https://github.com/python/pythontestdotnet which is used for testing purposes? Is it necessary to make a request to pypi since we can define a minimal custom dict based on the current example to show the output and usage of other parameters like `depth` and `width`

Reply:
By using PyPI, it looks more "pragmatic" rather than "artificial" example.

I found nice example: https://pypi.org/pypi/sampleproject/json
This project is maintained by PyPA.